core/sql/exp/ExpLOBaccess.cpp

```diff
@@ -2454,7 +2560,55 @@ Ex_Lob_Error ExLobsOper (
       err = lobPtr->writeDataSimple(source, sourceLen, subOperation, retOperLen,
 				    bufferSize , replication , blockSize);
       break;
+    case Lob_InsSel:
+      {
+        
+        ExLob *srcLobPtr;
+        Int16 flags;
+        Lng32  lobNum;
+        Int32 lobType;
+        Int64 uid, inDescSyskey, descPartnKey;
+        short schNameLen;
+        char schName[1024];
```

Comment:
I see several fixed-length char arrays here. Do we have guarantees that these won't overrun?